src/app/Main/Body/InfoPage/InfoPage.model.tsx

```diff
@@ -15,16 +15,26 @@ export interface AmrsTable {
     titleString: string;
     description: string;
     tableHeader: string[];
+    tableSubHeader: string[];
     tableRows: AmrsTableData[];
-    commentText: string;
 }
 
-export type AmrKey = "coliSalm" | "coliSalmTwo" | "campy" | "mrsa" | "ef";
+export type AmrKey =
```

Comment:
I recommend to not mix the two different types of table identifiers. You could see the tableX format as your internal translation identifier and use the old style as the armKey (coliSalm | coliSalmTwo ...)
Also, why did you introduced tableX format at all?

Reply:
Because there are now two tables for Campy, the old ArmKeys were no longer unique.

Reply:
So campy1 and campy2 would have done the job?
So think also about if you want to switch table orders, then you have to rename the tables and the translation definitions.
Just a suggestion, so you can leave it as it is.